src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -270,7 +325,43 @@ export const NewProposal = ({ navigation, data, query }) => {
   );
 };
 
+const ImageSelector = ({ control, imageUri, item, onChange, selectImage, value }) => {
+  const { buttonTitle, infoText, label, name } = item;
+
+  useEffect(() => {
+    // the `!value` control is used for editing the proposal
+    // the proposal contains an image, it is for showing the preview to the users
+    if (!value) {
+      onChange(imageUri);
+    }
+  }, [imageUri]);
+
+  return (
+    <>
+      <Input hidden label={label} control={control} name={name} value={value} />
+      <RegularText smallest placeholder>
+        {infoText}
+      </RegularText>
+
+      {value ? (
+        <WrapperRow center spaceBetween>
+          <Image source={{ uri: value }} style={styles.image} />
```

Comment:
why is `value` used here and not `imageUri`?

Reply:
`imageUri` does not return an empty value after selecting an image. It just changes itself. If I use `imageUri` here, the user will not be able to delete the image and select another image after selecting it. Or I'll have to delete the value of `onChange("")` plus the `imageUri`.

Reply:
0bc389056155a1878daf189d621c7353bcc70657